charts/beyla/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: beyla
-version: 1.1.1
-appVersion: 1.5.2
+version: 1.2.0
```

Comment:
I don't have a clear opinion here. Do updating a minor version of the `appVersion` deserve an update of the minor version of the chart?

Reply:
In https://github.com/grafana/helm-charts the script would just bump to 1.1.2, but I guess i doesn't matter much